src/Components/StartMenu.js

```diff
@@ -1,25 +1,129 @@
 import React from "react";
+import * as myConstants from "./Constants";
+import TextField from '@mui/material/TextField';
+import Stack from '@mui/material/Stack'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { createTheme } from '@mui/material/styles';
+import { ThemeProvider } from "@mui/material";
 
-function StartMenu (props) {
-    const [inputText, setInputText] = React.useState("3");
 
-    const handleInput = event => {
 
-        setInputText(event.target.value);
-      };
+function StartMenu(props) {
+  const [inputText, setInputText] = React.useState("3");
+  const [inputUsername, setInputUsername] = React.useState("Noname");
```

Comment:
Why have this set it should be empty and have a validation on start click for both lanes and username. The inputs should become red.